jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpTester.java

```diff
@@ -262,17 +262,25 @@ public static Response parseResponse(Input input) throws IOException
         return parseResponse(input, false);
     }
 
+    public static Response parseResponse(Input input, Response response) throws IOException
```

Comment:
While restoring `FileBufferedResponseHandlerTest` (from earlier versions of Jetty) this method in `HttpTester` was also restored.